<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarAddingWindow</class>
 <widget class="QDialog" name="CarAddingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление автомобиля в каталог</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 240, 239);
font: 600 8pt &quot;Montserrat&quot;;
color: rgb(55, 51, 53);</string>
  </property>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="mileageLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mileageLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>136</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>184</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость</string>
   </property>
  </widget>
  <widget class="QLabel" name="brandLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>226</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Марку</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>268</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Модель</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>94</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>178</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>262</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="topLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>382</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
margin-bottom: 20px;
font: 700 10pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>Введите новую информацию об автомобиле</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 9pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Добавить автомобиль</string>
   </property>
  </widget>
  <widget class="QLabel" name="fuelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Объем
топливного бака</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fuelLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batteryLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ёмкость
батареи</string>
   </property>
  </widget>
  <widget class="QLabel" name="hybridLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите тип
гибрида</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hybridTypeComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox 
{
	color: rgb(30, 36, 39);
	font: 600 9pt &quot;Montserrat&quot;;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}
QComboBox::drop-down
{
	border:none;
}
QComboBox::down-arrow
{
	image: url(:/rcs/free-icon-down-arrow-2985150.png);
	wigth: 30px;
	height:30px;
	margin-right: 5px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>1. Параллельный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Последовательный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="engineTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите тип
двигателя</string>
   </property>
  </widget>
  <widget class="QComboBox" name="engineTypeComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox 
{
	color: rgb(30, 36, 39);
	font: 600 9pt &quot;Montserrat&quot;;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}
QComboBox::drop-down
{
	border:none;
}
QComboBox::down-arrow
{
	image: url(:/rcs/free-icon-down-arrow-2985150.png);
	wigth: 30px;
	height:30px;
	margin-right: 5px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>1. Электрический</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. ДВС</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3. Гибридный</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #9AB6AC;
}
QPushButton{
	color: rgb(55, 51, 53);
	font: 600 9pt &quot;Montserrat&quot;;
	border-radius: 10px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}

</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #CC3329;
font: 700 9pt &quot;Montserrat&quot;;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
